conf/cassandra.yaml

```
@@ -1259,6 +1259,11 @@ server_encryption_options:
   # Set to a valid keystore if internode_encryption is dc, rack or all
   keystore: conf/.keystore
   keystore_password: cassandra
+  # During internode mTLS authentication, Inbound connections uses keystore, keystore_password containing server certificate
+  # for creating SSLContext and outbound connections use outbound_keystore & outbound_keystore_password with client certificates
+  # for creating SSLContext. By default, outbound keystore is set to keystore.
```

Updated the format (adjusted with the line width) and rephrased some sentences a bit. Please take a look. 
```suggestion
  # During internode mTLS authentication, inbound connections (acting as servers) use keystore, keystore_password 
  # containing server certificate to create SSLContext and 
  # outbound connections (acting as clients) use outbound_keystore & outbound_keystore_password with client certificates
  # to create SSLContext. By default, outbound_keystore is the same as keystore indicating mTLS is not enabled.
```